clients/githubrepo/githubrepo_suite_test.go

```diff
@@ -55,6 +59,14 @@ func skipIfTokenIsNot(t tokenType, msg string) {
 	}
 }
 
+func getGithubToken() string {
+	token := os.Getenv("GITHUB_TOKEN")
+	if token == "" {
+		token = os.Getenv("GITHUB_AUTH_TOKEN")
+	}
+	return token
+}
+
```

Comment:
not sure this is needed if reworking to use roundtripper.NewTransport

Reply:
Agreed, I took care of it.